### DISCUSSION TITLE
future ready talent

### DISCUSSION THREAD

**Discussion by Kittu-2208** [github/india#354] [Students]
I am unable to move forward to learning paths in future ready talent internship. Please help me.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Reply by MOHAMMED-USMAN1** [reply]
Hey there 
Same even for me
I am unable to complete an exercise in Self learning WordPress Website Creation 
Have u completed if yes 
Plz suggest how to complete it

**Comment by MOHAMMED-USMAN1** [comment]
> I am unable to move forward to learning paths in future ready talent internship. Please help me.
